cli_tools/common/utils/param/param_utils.go

```diff
@@ -71,8 +71,18 @@ func populateScratchBucketGcsPath(scratchBucketGcsPath *string, zone string, mgc
 		if err != nil {
 			return "", daisy.Errf("invalid scratch bucket GCS path %v", scratchBucketGcsPath)
 		}
+
 		if !scratchBucketCreator.IsBucketInProject(*project, scratchBucketName) {
-			return "", daisy.Errf("Scratch bucket `%v` is not in project `%v`", scratchBucketName, *project)
+			err := storageClient.DeleteGcsPath(*scratchBucketGcsPath)
```

Comment:
This will delete the whole bucket, which could have unforseen consequences in normal use cases. We should be deleting args.SourceFile instead.

Reply:
updated to delete the source file